src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAlone.java

```diff
@@ -27,6 +27,13 @@ void method2(java.util.HashSet<String> set) {
             put("polygene", "lubricants");
             put("alpha", "betical");
         }}; //NO violation
+
+        java.util.Map<String, String> map3 = new java.util.LinkedHashMap<String, String>() {{
+            put("Hello", "World");
+            put("first", "second");
+            put("polygene", "lubricants");
+        }{};
+        }; // violation
```

Comment:
please move comment to line 35 , where there is a violation,

Reply:
Fixed